Firmware/mmu2.cpp

```diff
@@ -283,7 +283,7 @@ bool MMU2::set_filament_type(uint8_t index, uint8_t type) {
     // cmd_arg = filamentType;
     // command(MMU_CMD_F0 + index);
 
-    manage_response(false, false); // true, true);
+    manage_response(false, false); // true, true); -- Comment: how is it possible for a filament type set to fail?
```

Comment:
_how is it possible for a filament type set to fail_
We'll rewrite this method once we merge the generic read/write operations on the MMU. Setting filament type, speeds, accelerations, stallguard, etc. will be done through a set of write commands (which can theoretically fail in case of writing into a read-only register, but not otherwise)

Reply:
True - though I guess I was mainly confused why one might want to park and cool down at this stage since I don't think the printer is heating yet at this time. Perhaps the commented arguments are a stale copy-paste?